packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-wrapper.tsx

```diff
@@ -0,0 +1,251 @@
+    is_virtual,
+    jurisdiction_selected_shortcode,
+    openPasswordModal,
+    real_financial_accounts_existing_data,
+    real_swapfree_accounts_existing_data,
+    real_synthetic_accounts_existing_data,
+    residence,
+    residence_list,
+    setJurisdictionSelectedShortcode,
+    should_restrict_bvi_account_creation,
+    should_restrict_vanuatu_account_creation,
+    show_eu_related_content,
+    toggleCFDVerificationModal,
+    trading_platform_available_accounts,
+    toggleJurisdictionModal,
+    updateMT5Status,
+}: TJurisdictionModalContentWrapperProps) => {
+    const [checked, setChecked] = React.useState(false);
+
+    const {
+        poi_or_poa_not_submitted,
+        poi_acknowledged_for_bvi_labuan,
+        poi_acknowledged_for_vanuatu_maltainvest,
+        poa_acknowledged,
+        need_poa_resubmission,
+    } = getAuthenticationStatusInfo(account_status);
+
+    React.useEffect(() => {
+        if (is_jurisdiction_modal_visible) {
+            if (!is_virtual) {
+                updateMT5Status();
+                fetchAccountSettings();
+            }
+            setJurisdictionSelectedShortcode('');
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [is_jurisdiction_modal_visible]);
+
+    React.useEffect(() => {
+        if (jurisdiction_selected_shortcode) {
+            setChecked(false);
+        }
+    }, [jurisdiction_selected_shortcode, is_jurisdiction_modal_visible]);
+
+    const financial_available_accounts = trading_platform_available_accounts.filter(
+        available_account =>
+            available_account.market_type === 'financial' &&
+            (show_eu_related_content
+                ? available_account.shortcode === 'maltainvest'
+                : available_account.shortcode !== 'maltainvest')
+    );
+
+    const synthetic_available_accounts = trading_platform_available_accounts.filter(
+        available_account =>
+            available_account.market_type === 'gaming' &&
+            (show_eu_related_content
+                ? available_account.shortcode === 'maltainvest'
+                : available_account.shortcode !== 'maltainvest')
+    );
+
+    const is_svg_selected = jurisdiction_selected_shortcode === Jurisdiction.SVG;
+    const is_bvi_selected = jurisdiction_selected_shortcode === Jurisdiction.BVI;
+    const is_vanuatu_selected = jurisdiction_selected_shortcode === Jurisdiction.VANUATU;
+    const is_labuan_selected = jurisdiction_selected_shortcode === Jurisdiction.LABUAN;
+    const is_maltainvest_selected = jurisdiction_selected_shortcode === Jurisdiction.MALTA_INVEST;
+
+    const is_idv_country =
+        residence_list.find(elem => elem?.value === residence)?.identity?.services?.idv?.is_country_supported === 1;
+    const has_idv_attempts = (account_status?.authentication?.identity?.services?.idv?.submissions_left ?? 0) > 0;
+    const is_onfido_design = !is_idv_country || (is_idv_country && !has_idv_attempts);
```

Comment:
could you please explain the logic behind this? we caanot say a country is supported by onfido based on idv_supported countries list. There are countries where both idv and onfido are not supported.

Reply:
@amina-deriv i am displaying the onfido design by checking the following conditions. if not onfido, we will fallback to idv card design. Please let me know if this logic need to be changed.
![Screenshot 2023-05-29 at 5 33 56 PM](https://github.com/binary-com/deriv-app/assets/122449658/81c65172-e005-495e-a149-788fb7b58b1d)

Reply:
we have to check if the country is supported by onfido before displaying onfido design. Not all countries support onfido. There are countries which doesnt support both idv and onfido in which we display manual directly,
you can check residence_list-> country-> services->onfido->is_country_supported

Reply:
we have to check if the country is supported by onfido before displaying onfido design. Not all countries support onfido. There are countries which doesnt support both idv and onfido in which we display manual directly,
you can check residence_list-> country-> services->onfido->is_country_supported

Reply:
@amina-deriv as discussed, i have renamed the variable 'is_onfido_design' to 'is_non_idv_design' to avoid ambiguity.